src/EditorFeatures/TestUtilities/StubVsServiceExporter`2.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.ComponentModel.Composition;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.VisualStudio;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+using Microsoft.VisualStudio.Threading;
+
+// Import Roslyn.Utilities with an alias to avoid conflicts with AsyncLazy<T>. This implementation relies on
+// AsyncLazy<T> from vs-threading, and not the one from Roslyn.
+using RoslynUtilities = Roslyn.Utilities;
```

Comment:
Is it easier to just `using SpecializedTasks = Roslyn.Utilities.SpecializedTasks`? The there's not the odd qualification later. The comment though was helpful, so definitely leave that!